nsxt/resource_nsxt_policy_lb_http_application_profile.go

```diff
@@ -0,0 +1,249 @@
+				Description: "A URL that incoming requests for that virtual server can be temporarily redirected to, If a website is temporarily down or has moved",
+				Optional:    true,
+			},
+			"http_redirect_to_https": {
+				Type:        schema.TypeBool,
+				Description: "A boolean flag which reflects whether the client will automatically be redirected to use SSL",
+				Optional:    true,
+				Default:     false,
+			},
+			"idle_timeout": {
+				Type:         schema.TypeInt,
+				Description:  "Timeout in seconds to specify how long an HTTP application can remain idle",
+				Optional:     true,
+				Default:      15,
+				ValidateFunc: validation.IntAtLeast(1),
+			},
+			"ntlm": {
+				Type:        schema.TypeBool,
+				Description: "A boolean flag which reflects whether NTLM challenge/response methodology will be used over HTTP",
+				Optional:    true,
+				Default:     false,
+			},
+			"request_body_size": {
+				Type:        schema.TypeInt,
+				Description: "Maximum request body size in bytes (Unlimited if not specified)",
+				Optional:    true,
+			},
+			"request_header_size": {
+				Type:         schema.TypeInt,
+				Description:  "Maximum request header size in bytes. Requests with larger header size will be processed as best effort whereas a request with header below this specified size is guaranteed to be processed",
+				Optional:     true,
+				Default:      1024,
+				ValidateFunc: validation.IntBetween(1, 65536),
+			},
+			"response_buffering": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "A boolean flag indicating whether the response received by LB from the backend will be saved into the buffers",
+				Default:     false,
+			},
+			"response_header_size": {
+				Type:         schema.TypeInt,
+				Description:  "Maximum request header size in bytes. Requests with larger header size will be processed as best effort whereas a request with header below this specified size is guaranteed to be processed",
+				Optional:     true,
+				Default:      4096,
+				ValidateFunc: validation.IntBetween(1, 65536),
+			},
+			"response_timeout": {
+				Type:         schema.TypeInt,
+				Description:  "Number of seconds waiting for the server response before the connection is closed",
+				Optional:     true,
+				Default:      60,
+				ValidateFunc: validation.IntAtLeast(1),
+			},
+			"server_keep_alive": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "A boolean flag indicating whether the backend connection will be kept alive for client connection. If server_keep_alive is true, it means the backend connection will keep alive for the client connection. Every client connection is tied 1:1 with the corresponding server-side connection. If server_keep_alive is false, it means the backend connection won't keep alive for the client connection. If server_keep_alive is not specified for API input, its value in API output will be the same with the property ntlm",
```

Comment:
The last sentence is not relevant for the provider, since its always provides a value - lets remove it